subsys/lorawan/Kconfig

```diff
@@ -0,0 +1,68 @@
+# LoRaWAN configuration options
+
+# Copyright (c) 2020 Manivannan Sadhasivam <mani@kernel.org>
+# SPDX-License-Identifier: Apache-2.0
+
+menuconfig LORAWAN
+	bool "LoRaWAN support [EXPERIMENTAL]"
+	depends on LORA
+	select HAS_SEMTECH_LORAMAC
+	select HAS_SEMTECH_SOFT_SE
+	help
+	  This option enables LoRaWAN support.
+
+if LORAWAN
+
+module = LORAWAN
+module-str = lorawan
+source "subsys/logging/Kconfig.template.log_config"
+
+config LORAWAN_SYSTEM_MAX_RX_ERROR
+	int "LoRaWAN System Max Rx Error"
+	default 20
+	help
+	  System Max Rx timing error value in ms to be used by LoRaWAN stack
+	  for calculating the RX1/RX2 window timing.
+
+choice
+	prompt "LoRaWAN Region Selection"
+	default LORAMAC_REGION_UNKNOWN
+	help
+	  Select the LoRaWAN region.
+
+config LORAMAC_REGION_UNKNOWN
```

Comment:
Just to understand, by "choice" configuration, you have not the possibility to dynamically change region. Am I right ?

Reply:
Yes, that's correct! The region setting is static.